### PR TITLE
Fix sourcemaps

### PR DESCRIPTION
**Pull request by chrisprobably**

### CODE REVIEW

**Comment by chrisprobably** [comment]
To enable sourcemaps you need to pass the command line parameter to the server `node server/node/server.js --sourceMaps`

File: webpack-config-app/babel.js [hunk 1]
```diff
@@ -113,5 +113,6 @@ module.exports = function configureBabelLoader(webpackConfig, basePath) {
     options: createBabelLoaderOptions(basePath)
   };
 
-  webpackConfig.module.rules.push(babelModulesRule);
+  // Babel loader must be first for source maps to be shown in ES6
```
Comment (briandipalma) [inline webpack-config-app/babel.js]: Any chance you could add a link to the issue in GitHub?

https://github.com/webpack-contrib/transform-loader/issues/9

Comment (chrisprobably) [inline webpack-config-app/babel.js]: Done

**Comment by briandipalma** [comment]
Published new version.

**Pull request merged**